staging/src/k8s.io/cri-api/pkg/apis/runtime/v1alpha2/api.proto

```diff
@@ -279,13 +279,29 @@ message LinuxSandboxSecurityContext {
     // This allows a sandbox to take additional security precautions if no
     // privileged containers are expected to be run.
     bool privileged = 6;
-    // Seccomp profile for the sandbox, candidate values are:
-    // * runtime/default: the default profile for the container runtime
-    // * unconfined: unconfined profile, ie, no seccomp sandboxing
-    // * localhost/<full-path-to-profile>: the profile installed on the node.
-    //   <full-path-to-profile> is the full path of the profile.
-    // Default: "", which is identical with unconfined.
-    string seccomp_profile_path = 7;
```

Comment:
We need to do this in 2 steps so container runtimes could be updated to use the new field.

Reply:
Yes that's why the field is now deprecated. I think this just implies a `Deprecated: Do not use` comment in the actual go code but then linters can hint to it.